torchprime/torch_xla_models/README.md

```diff
@@ -14,12 +14,14 @@ These models use the [torch_xla][1] framework.
    export XLA_HLO_DEBUG=1
    ```
 
-1. Run the trainer. The default is to train Llama 3.0 8B sharded over 4 chips.
+1. Run the trainer. The default is to train Llama 3.0 8B sharded over 4 v6e chips.
```

Comment:
nit: technically it doesn't assume that the VM is v6e. Maybe rephrase as "The default config trains Llama 3.0 8B sharded over 4 chips and is tested for v6e".

Reply:
Updated, thanks.

Reply:
SGTM